README

```diff
@@ -12,6 +12,6 @@ While it is meant to be used as a library, a simple command-line tool is
 provided as an example. It operates on RAW 16-bit (machine endian) mono
 PCM files sampled at 48 kHz. It can be used as:
 
-./examples/rnnoise_demo <number of channels> <maximum attenuation> < input.raw > output.raw
+./examples/rnnoise_demo <input.raw> output.raw
```

Comment:
You could also just use the same form as the `usage` message which I think is a bit clearer:
```
<noisy speech> <output denoised>
```

See: https://github.com/xiph/rnnoise/blob/125d8a56e0049728c86c9a575dab348fc9523e96/examples/rnnoise_demo.c#L41

Reply:
Changed to using the same form as the `usage` message :)